javascriptv3/example_code/cross-services/ddb-item-tracker/README.md

```diff
@@ -0,0 +1,729 @@
+4. An Amazon S3 bucket (this is used to store the CSV file)
+5. Two email addresses verified in Amazon SES
+
+
+You can create the AWS resources required for this cross-service example using either of the following:
+- [Option 1: Amazon CloudFormation](#create-the-resources-using-amazon-cloudformation)
+- [Option 2: The AWS Management Console](#create-the-resources-using-the-aws-management-console)
+
+#### <a name="create-the-resources-using-amazon-cloudformation"></a>Option 1: Create the resources using Amazon CloudFormation
+
+You can run a single Amazon CloudFormation script to provision all the required resources for this example.
+
+1. Install and configure the AWS CLI following the instructions in the AWS CLI User Guide.
+
+2. Open the AWS Command Console from the *./ddb-item-tracker* folder.
+
+3. Run the following command, replacing *STACK_NAME* with a unique name for the stack.
+```
+aws cloudformation create-stack --stack-name STACK_NAME --template-body file://setup.yaml --capabilities CAPABILITY_IAM
+```
+**Important**: The stack name must be unique within an AWS Region and AWS account. You can specify up to 128 characters, and numbers and hyphens are allowed.
+
+4. Open [AWS CloudFormation in the AWS Management Console](https://aws.amazon.com/cloudformation/), and open the **Stacks** page, and chose the **Resources** tab.
+
+![ ](images/cloud_formation_stacks.png)
+
+5. Choose the **Resources** tab. The resources required for this cross-service example are listed in the **Physical ID** column.
+
+![ ](images/cloud_formation_resources_tab.png)
+
+For more information on the create-stack command parameters, see the [AWS CLI Command Reference guide](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/create-stack.html), and the [AWS CloudFormation User Guide](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-cli-creating-stack.html).
+
+#### <a name="create-the-resources-using-the-aws-management-console"></a>Option 2: Create the resources using the AWS Management Console
+
+#### 1. Create an unauthenticated user role
+1. Open [AWS Cognito in the AWS Management Console](https://aws.amazon.com/cloudformation/), and open the *Stacks* page.
+2. Choose **Manage Identity Pools**.
+3. Choose **Create new identity pool**.
+4. In the **Identity pool name** field, give your identity pool a name.
+5. Select the **Enable access to unauthenticated identities** checkbox.
+6. Choose **Create Pool**.
+7. Choose **Allow**.
+8. Take note of the **Identity pool ID**, which is highlighted in red in the **Get AWS Credentials** section.
+
+![ ](images/identity_pool_ids.png)
+
+9.Choose **Edit identity pool**.
+10. Take note of the name of the role in the **Unauthenticated role** field.
+
+#### 2. Adding permissions to an unauthenticated user role
+1. Open [IAM in the AWS Management Console](https://aws.amazon.com/iam/), and open the *Roles* page.
```

Comment:
Did you mean this to be a link to the console? It's to an IAM info page.

Reply:
done